cli/features/support/world.ls

```diff
@@ -13,6 +13,11 @@ World = !->
     fs.copy-sync path.join(example-apps-path, app-name), app-dir
 
 
+  @checkout-service-template = (app-name, template-name) ->
+    template-dir = path.join process.cwd!, 'tmp', app-name, '.exosphere', template-name
+    fs.copy-sync path.join(process.cwd!, '..' 'exosphere-shared' 'templates', 'boilr-templates', template-name), template-dir
```

Comment:
thoughts on updating this to
```livescript
src = #...
dest = #....
fs.copy-sync src, dest
```
Having the src inlined makes this very hard for me to read